pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsctl"
+	"github.com/vmware-tanzu/antrea/pkg/util/ip"
+)
+
+// connTrackOvsCtl implements ConnTrackDumper. This supports OVS userspace datapath scenarios.
+var _ ConnTrackDumper = new(connTrackOvsCtl)
+
+type connTrackOvsCtl struct {
```

Comment:
do you think you can split this file into: conntrack.go (common stuff) and conntrack_ovs.go?

Reply:
Done.

Reply:
Done